### PR TITLE
add the ability to load all customizations from a config file.

### PR DESCRIPTION
**Pull request by AntonioMeireles**
(the _real_ reason for all the previous commits :smile:). currently on top of #29.

### CODE REVIEW

**Comment by rimusz** [comment]
@AntonioMeireles you just rock mate :)
now we are just left with attached storage problem

**Comment by rimusz** [comment]
@AntonioMeireles question about cloud-config, can we just point to some cloud-config to use some local file? as using OS X apache it is not always handy

**Comment by AntonioMeireles** [comment]
afaik i needs to be a valid (http) URL (as that is what CoreOS cloud-config machinery expects...). doesn't need to be apple's httpd. any webserver will do. fwiw it crossed my mind before to ship a small golang blob that would just http-serve the cloud-config but that looked to me as way over-engineering.

**Comment by bakins** [comment]
LGTM taking into account my comments on `SSHKEY` on #29

File: coreos-xhyve-run [hunk 1]
```diff
@@ -1,4 +1,18 @@
 #!/bin/bash
+OPTIND=1
+while getopts ":f:" opt; do
+  case ${opt} in
+    f)
+      [[ -r "${OPTARG}" ]] && source "${OPTARG}" || \
+        echo "Warning: couldn't read \"${OPTARG}\". Ignoring." >&2
```
Comment (bakins) [inline coreos-xhyve-run:7]: should this be a fatal error?

Comment (AntonioMeireles) [inline coreos-xhyve-run:7]: @bakins i opted for not making it fatal for 'politeness'. matter of taste i suppose. up to you :smile:

meanwhile just rebased previous PR against upstream.

**Pull request merged**